BackEnd/src/main/java/com/javaschool/entitymapper/MessageType.java

```diff
@@ -0,0 +1,11 @@
+package com.javaschool.entitymapper;
+
```

Comment:
tip: Take a look to lombok library to avoid boilerplate code.
[https://projectlombok.org](url)